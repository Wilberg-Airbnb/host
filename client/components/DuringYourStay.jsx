import React from 'react';
import { Item } from 'semantic-ui-react';

const DuringYourStay = ({duringYourStay}) => (
  <Item>
    <Item.Content>
        <h5>During your stay</h5>
      <Item.Description>
      {duringYourStay}
      </Item.Description>
    </Item.Content>
  </Item>
)

export default DuringYourStay;