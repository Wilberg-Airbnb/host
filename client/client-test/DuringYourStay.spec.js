import React from 'react';
import DuringYourStay from '../components/DuringYourStay.jsx';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
 
configure({ adapter: new Adapter() });
 
describe('Renders Components', () => {
  let component;
  let prop = 'Commodi perspiciatis ullam et quod ipsa est porro. Ratione maiores rerum soluta sed. Dolores adipisci rerum sed omnis distinctio et facere cupiditate. Ut deserunt quo non praesentium id. Excepturi itaque adipisci consectetur voluptatibus. Sequi consequatur voluptatum et dolor.'
  beforeEach( () => {
    component = shallow(<DuringYourStay duringYourStay={prop}/>)
  });

  test("should render initial layout", () => {
    const component = shallow(<DuringYourStay duringYourStay={prop} />);
    expect(component.getElements()).toMatchSnapshot();
  });

});


