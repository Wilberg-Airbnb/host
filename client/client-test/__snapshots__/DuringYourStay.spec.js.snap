// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Components should render initial layout 1`] = `
Array [
  <Item>
    <ItemContent>
      <h5>
        During your stay
      </h5>
      <ItemDescription>
        Commodi perspiciatis ullam et quod ipsa est porro. Ratione maiores rerum soluta sed. Dolores adipisci rerum sed omnis distinctio et facere cupiditate. Ut deserunt quo non praesentium id. Excepturi itaque adipisci consectetur voluptatibus. Sequi consequatur voluptatum et dolor.
      </ItemDescription>
    </ItemContent>
  </Item>,
]
`;
